src/components/notifications/GtxMessaging.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+    <div class="mb-3">
+        <label for="gtxmessaging-api-key" class="form-label">{{ $t("gtxMessagingApiKey") }}</label>
+        <HiddenInput id="gtxmessaging-api-key" v-model="$parent.notification.gtxMessagingApiKey" :required="true"></HiddenInput>
+    </div>
+    <div class="mb-3">
+        <label for="gtxmessaging-from" class="form-label">{{ $t("gtxMessagingFrom") }}</label>
```

Comment:
please either choose a generic name for this field like `"From Name/Number"` or be more specific.

Reply:
Applied wording from gtx-messaging site